### PR TITLE
Fix RWAHS-443 Copy `groupAccess` from ui into derived displays

### PR DESCRIPTION
**Pull request by kehh**
* Change xslt to copy these access permissions
* Update profile with these new derived values
* Also add missing groupAccess for the `subject_list_ui`

### CODE REVIEW

File: profile/rwahs.xml [hunk 1]
```diff
@@ -6116,6 +6116,9 @@
       <groupAccess>
         <permission group="admin" access="edit"/>
         <permission group="museum" access="edit"/>
+        <permission group="library" access="edit"/>
```
Comment (kehh) [inline profile/rwahs.xml]: These 3 lines were missing, as all roles have access to managing the subject list.

File: profile/rwahs.xml [hunk 2]
```diff
@@ -8063,6 +8066,13 @@
       <typeRestrictions>
         <restriction type="library"/>
       </typeRestrictions>
+      <groupAccess>
```
Comment (kehh) [inline profile/rwahs.xml]: This is copied from [`library_object_ui`](https://github.com/kehh/installation-profile/blob/9c15cd22a68ee49d4e5d43d10deae7045426412e/profile/rwahs.xml#L5358-L5364) using the updated `derive-displays.xslt` below:

File: profile/rwahs.xml [hunk 3]
```diff
@@ -8329,6 +8339,13 @@
       <typeRestrictions>
         <restriction type="publicMemorial"/>
       </typeRestrictions>
+      <groupAccess>
```
Comment (kehh) [inline profile/rwahs.xml]: This is copied from [`memorial_object_ui`](https://github.com/kehh/installation-profile/blob/9c15cd22a68ee49d4e5d43d10deae7045426412e/profile/rwahs.xml#L5880-L5886) using the updated `derive-displays.xslt` below.

File: profile/rwahs.xml [hunk 4]
```diff
@@ -9257,6 +9288,13 @@
       <typeRestrictions>
         <restriction type="Subject"/>
       </typeRestrictions>
+      <groupAccess>
```
Comment (kehh) [inline profile/rwahs.xml]: This one contains the new values from the subject UI access above.

File: profile/rwahs.xml [hunk 5]
```diff
@@ -8462,6 +8479,13 @@
       <typeRestrictions>
         <restriction type="museum"/>
       </typeRestrictions>
+      <groupAccess>
```
Comment (kehh) [inline profile/rwahs.xml]: as above

File: profile/rwahs.xml [hunk 6]
```diff
@@ -8905,6 +8929,13 @@
       <typeRestrictions>
         <restriction type="photograph"/>
       </typeRestrictions>
+      <groupAccess>
```
Comment (kehh) [inline profile/rwahs.xml]: as above

File: profile/rwahs.xml [hunk 7]
```diff
@@ -9300,6 +9338,10 @@
       <typeRestrictions>
         <restriction type="Conservation"/>
       </typeRestrictions>
+      <groupAccess>
```
Comment (kehh) [inline profile/rwahs.xml]: Copied from [`conservation_ui`] (https://github.com/kehh/installation-profile/blob/9c15cd22a68ee49d4e5d43d10deae7045426412e/profile/rwahs.xml#L6181-L6184)

File: transforms/derive-displays.xslt [hunk 8]
```diff
@@ -123,6 +123,7 @@
         </xsl:element>
       </xsl:element>
       <xsl:copy-of select="/profile/userInterfaces/userInterface[@code = $ui_code]/typeRestrictions" />
+      <xsl:copy-of select="/profile/userInterfaces/userInterface[@code = $ui_code]/groupAccess" />
```
Comment (kehh) [inline transforms/derive-displays.xslt]: And this is the actual change that generated most of the above changes ;)

**Comment by mostro20** [comment]
First time I've reviewed a XSLT file - very readable... Enjoyed the comments :-)

**Comment by mostro20** [comment]
In the XSLT, would be interested in finding out what the test was doing around line 145 with "emplate" as I could follow what was happening in the rest, but got a bit lost there - maybe during the stand-up... just out of interest more than anything else.

**Comment by mostro20** [comment]
Can see the relationship between the XSLT file changes and the XML output. Everything seems to be in order.

**Pull request merged**